src/main/java/it/gov/pagopa/payhub/auth/common/config/SwaggerConfig.java

```diff
@@ -0,0 +1,35 @@
+package it.gov.pagopa.payhub.auth.common.config;
```

Comment:
the "common" package is not inside "auth" module
```suggestion
package it.gov.pagopa.payhub.common.config;
```